src/main/resources/org/apache/brooklyn/tosca/a4c/custom/types/custom-types.yml

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Custom types. Currently, Tomcat is converted to a `TomcatServer` entity and ComputeLoc to a `VanillaSoftwareProcess` which contains a location.
As I mentioned, I think brooklyn entities should be added to this custom types